css/component_styles/signin.css

```diff
@@ -7,4 +7,58 @@
 body {
     background-color: #ffffff;
     background-image: url("../../assets/bg.svg");
-}
\ No newline at end of file
+}
+
+.login-signup-content {
+    display: grid;
+    place-items: center;
+}
+
+.login-signup-form {
+    background-color: var(--base-color);
+    border-radius: var(--border-5);
+    padding: var(--size-1r);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    box-shadow: var(--card-shadow);
+    min-height: 30rem;
+    min-width: 30rem;
+}
+
+.login-signup-form>* {
```

Comment:
could add a space before >* to make it more visible

Reply:
ok i will do it